### PR TITLE
#3308 Fix claimed token not showing in assets

### PR DESCRIPTION
**Pull request by drakkein**
I did small tweak to existing code. It seems to be updating correctly. 

Video: https://imgur.com/a/0iHunkb

### CODE REVIEW

File: packages/web/app/modules/tx/transactions/claim/sagas.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ function* userClaimSaga(
     logger.info("User claim cancelled", e);
   } finally {
     yield put(actions.eto.loadEto(etoId));
-    yield put(actions.eto.loadTokensData());
+    yield put(actions.eto.loadTokenData(etoId));
```
Comment (drakkein) [inline packages/web/app/modules/tx/transactions/claim/sagas.ts]: The only change is that we are not relaying on `selectMyAssets` and updating all tokens data, but specified one.

File: packages/web/app/modules/tx/transactions/claim/sagas.ts [hunk 2]
```diff
@@ -76,7 +76,6 @@ function* userClaimSaga(
     logger.info("User claim cancelled", e);
   } finally {
     yield put(actions.eto.loadEto(etoId));
-    yield put(actions.eto.loadTokensData());
```
Comment (Moejoe90) [inline packages/web/app/modules/tx/transactions/claim/sagas.ts]: :+1:

File: packages/web/app/test-e2e/kyc/KYCFlowSupportedInstandIdProviders.spec.e2e.io.ts [hunk 3]
```diff
@@ -9,7 +9,8 @@ import {
 } from "./utils";
 
 describe("KYC flow supported instant id providers", () => {
-  it("should show a notification when none providers supported for the individual nationality is Vatican", () => {
+  // Onfido is supported for every country right now, these tests should be disabled
```
Comment (Moejoe90) [inline packages/web/app/test-e2e/kyc/KYCFlowSupportedInstandIdProviders.spec.e2e.io.ts]: :1st_place_medal:

File: packages/web/app/modules/eto/sagas.ts [hunk 4]
```diff
@@ -505,76 +509,111 @@ function* downloadTemplateByType(
...
-    const tokenControllerMe: ITokenController = yield contractsService.getTokenController(
-      tokenController,
-    );
-    const canTransferToken = yield tokenControllerMe.onTransfer(
-      walletAddress,
-      walletAddress,
-      generateRandomEthereumAddress(),
-      new BigNumber("1"),
-    );
+function* loadTokenData(_: TGlobalDependencies): Generator<any, any, any> {
```
Comment (Moejoe90) [inline packages/web/app/modules/eto/sagas.ts]: please type the `Generator`

File: packages/web/app/modules/eto/sagas.ts [hunk 5]
```diff
@@ -505,76 +509,111 @@ function* downloadTemplateByType(
   }
 }
 
-function* loadTokensData({ contractsService }: TGlobalDependencies): any {
+function* loadToken(
+  { contractsService }: TGlobalDependencies,
+  eto: TEtoWithCompanyAndContractReadonly,
+): Generator<any, any, any> {
```
Comment (Moejoe90) [inline packages/web/app/modules/eto/sagas.ts]: Please type the generator

**Pull request merged**